modules/prometheus-cpp/1.2.4/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
```

Comment:
Could you please also add a test with --config=ssl

Reply:
Ok, it's done :) !

Reply:
You'll have to use a test module to test with `--config=ssl`, otherwise, the presubmit creates a dummy module that depends on this module, and there won't be `ssl` in the bazelrc file.

Reply:
See https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#anonymous-module-test

Reply:
This needs to be added inside prometheus-cpp project or can it be done throught a patch for example ?

Reply:
I think the easiest way is to specify the `module_path` as `.`, then you can build as if the dependency itself is the root module. Then you can build `//core` and `//pull`.

The other way is to directly add [--define=with_civetweb_ssl=true](https://github.com/jupp0r/prometheus-cpp/blob/284a2e237960f48f34f0bb6167fdcf2bd3a027cf/.bazelrc#L10) as a build option